pandas/core/indexes/datetimes.py

```diff
@@ -1904,24 +1904,47 @@ def delete(self, loc):
 
     def tz_convert(self, tz):
         """
-        Convert tz-aware DatetimeIndex from one time zone to another (using
-        pytz/dateutil)
+        Convert tz(timezone)-aware DatetimeIndex using pytz/dateutil.
+
+        When using DatetimeIndex providing with timezone this method
+        converts tz-aware DatetimeIndex from one timezone to another.
 
         Parameters
         ----------
         tz : string, pytz.timezone, dateutil.tz.tzfile or None
-            Time zone for time. Corresponding timestamps would be converted to
-            time zone of the TimeSeries.
+            Time zone for time.Corresponding timestamps would be converted
+            to time zone of the TimeSeries.
             None will remove timezone holding UTC time.
 
         Returns
         -------
         normalized : DatetimeIndex
 
         Raises
-        ------
+        -------
         TypeError
             If DatetimeIndex is tz-naive.
+
+        See Also
+        --------
+        tz_localize : Localize tz-naive DatetimeIndex to given time zone
+            (using pytz/dateutil),or remove timezone from tz-aware
+            DatetimeIndex.
+
+        Examples
+        --------
+        >>> datetime=pd.Series(pd.date_range('20180301',periods=3))
+        >>> datetime
+        0   2018-03-01
+        1   2018-03-02
+        2   2018-03-03
+        dtype: datetime64[ns]
+
+        >>> datetime.dt.tz_localize('UTC').dt.tz_convert('US/Eastern')
+        0   2018-02-28 19:00:00-05:00
+        1   2018-03-01 19:00:00-05:00
+        2   2018-03-02 19:00:00-05:00
+        dtype: datetime64[ns, US/Eastern]
```

Comment:
Can you also add an example that uses `tz_convert(None)` ? 
It's explained at the end of this section: https://pandas.pydata.org/pandas-docs/stable/timeseries.html#working-with-time-zones